librosa/feature/spectral.py

```diff
@@ -161,7 +161,12 @@ def spectral_centroid(
     >>> ax.legend(loc='upper right')
     >>> ax.set(title='log Power spectrogram')
     """
-
+    
+    #input is time domain:y or spectrogam:s
```

Comment:
remove these comments and empty lines?

Reply:
I'm not too worried about empty lines / formatting right now.  We'll `black` the code before merging into main.